src/Enhavo/Bundle/AppBundle/Resources/config/services/services.yml

```diff
@@ -84,3 +87,8 @@ services:
         class: Enhavo\Bundle\AppBundle\Template\ConfigWebpackBuildResolver
         arguments:
             - '%enhavo_app.webpack_build%'
+
+    enhavo_app.argument_resolver:
```

Comment:
Please use the FQCN as service id as this style is old and not longer recommended in symfony.

```yaml
    Enhavo\Bundle\AppBundle\Controller\ArgumentResolver:
        arguments:
            - '@argument_metadata_factory'
```